CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.6.6] - 2021-12-10
+
+### Changed
+
+- update supported k8s version to 1.22 (#75)
```

Comment:
It is good to write the license change.
https://github.com/cybozu-go/contour-plus/pull/73

Reply:
I don't care write it or not, but other software don't state license change in CHANGELOG.

What do you think?

Reply:
In my opinion, the license change affects the users. So it is better to write it.